src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.OSX/AppleCertificatePal.cs

```diff
@@ -90,62 +92,48 @@ public static ICertificatePal FromOtherCert(X509Certificate cert)
                 throw new CryptographicException(SR.Cryptography_X509_PKCS7_NoSigner);
             }
 
-            bool exportable = true;
-
-            SafeKeychainHandle keychain;
-
             if (contentType == X509ContentType.Pkcs12)
             {
                 if ((keyStorageFlags & X509KeyStorageFlags.EphemeralKeySet) == X509KeyStorageFlags.EphemeralKeySet)
                 {
                     throw new PlatformNotSupportedException(SR.Cryptography_X509_NoEphemeralPfx);
                 }
 
-                exportable = (keyStorageFlags & X509KeyStorageFlags.Exportable) == X509KeyStorageFlags.Exportable;
+                bool exportable = (keyStorageFlags & X509KeyStorageFlags.Exportable) == X509KeyStorageFlags.Exportable;
```

Comment:
HasFlag?

Reply:
Now that HasFlag has been improved (it used to be terrible perf) it'd be reasonable to use it, but there's not really a logical change associated with the line (just changing the variable scope), so changing to use HasFlag here is a style-without-substance change below my personal bar.

If you wanted to change all the flags tests to HasFlag on a PR that didn't also have functional changes, that'd be fine :smile:, I just don't like mixing functional and style.